src/main/java/functions/Constants.kt

```diff
@@ -0,0 +1,5 @@
+package functions
+
+import tree.TargetTreeNode
+
+typealias VisitorFunction = (TargetTreeNode) -> List<TargetTreeNode>
```

Comment:
This is awesome! 😎 
Super Clean!
![](https://media.giphy.com/media/HIrYP4RI9DpLy/giphy.gif)

Reply:
![](https://media.giphy.com/media/S6HEz4XChX8boa5uuo/giphy.gif)